README.md

```diff
@@ -1,7 +1,19 @@
-# Shortest
-**[shortest.com](https://shortest.com)**
 
-AI-powered natural language end-to-end testing framework.
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/0adf7c5d-296f-46a5-8071-57c408f02d43" alt="Shortest Logo" width="200" />
```

Comment:
I think we should use an SVG to mirror the app, or it doesn't work? Also, should be orange, the hex code equivalent of `text-orange-500`